tsdb/shard_test.go

```diff
@@ -48,9 +48,6 @@ func TestShardWriteAndIndex(t *testing.T) {
 	}
 
 	validateIndex := func() {
-		if !reflect.DeepEqual(index.Names(), []string{"cpu"}) {
```

Comment:
Isn't this important test coverage? Can we replicate it another way? Seems like we could by comparing the expected measurement names with those in `measurements` map.